src/chapters/strings/string-methods.rst

```diff
@@ -0,0 +1,113 @@
+.. _string-methods:
+
+String Methods
+==============
+
+JavaScript provides many useful methods for string objects. Recall that a method is a function that "belongs to" a specific object. Methods will typically result in some operation being carried out on the data within an object. For strings, this means that our methods will typically transform the characters of the given string in some way.
+
+As we have learned, strings are immutable. Therefore, string methods will not change the value of a string itself, but instead will *return* a new string that is the result of the given operation.
+
+We saw this behavior in the ``toLowerCase`` example.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+   
+      let nonprofit = "LaunchCode";
+
+      console.log(nonprofit.toLowerCase());
+      console.log(nonprofit);
+
+   **Output**
+
+   ::
+
+      launchcode
+      LaunchCode
+
+While ``nonprofit.toLowerCase()`` evaluated to ``"launchcode"``, the value of ``nonprofit`` was left unchanged. This will be case for each of the string methods.
+
+Common String Methods
+---------------------
+
+.. index::
+   single: string; methods
+
+Here we present the most commonly-used string methods. You can find documentation for other string methods at:
+
+- `W3Schools <https://www.w3schools.com/jsref/jsref_obj_string.asp>`_
+- `MDN <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#Methods_2>`_
+
+
+.. list-table:: Common String Methods
+   :header-rows: 1
+
+   * - Method
+     - Syntax
+     - Description
+   * - :ref:`indexOf <string-indexof-examples>`
+     - ``stringName.indexOf(character)``
+     - Returns the index of the first occurence of the character in the string, and returns -1 if the character is not found.
+   * - :ref:`toLowerCase <string-tolowercase-examples>`
+     - ``stringName.toLowerCase()``
+     - Returns a copy of the given string, with all uppercase letters converted to lowercase.
+   * - :ref:`toUpperCase <string-touppercase-examples>`
+     - ``stringName.toUpperCase()``
+     - Returns a copy of the given string, with all lowercase letters converted to uppercase.
+   * - :ref:`trim <string-trim-examples>`
+     - ``stringName.trim()``
+     - Returns a copy of the given string with a leading and trailing whitespace removed.
+   * - :ref:`replace <string-replace-examples>`
+     - ``stringName.replace(searchChar, replacementChar)``
+     - Given a search string ``searchChar`` and a replacement value ``replacementChar``, replace the first occurence of ``searchChar`` replaced by ``replacementChar``.
```

Comment:
Word reduction: The description sounds OK with just, "Replaces the first occurence of ``searchChar`` with ``replacementChar``."

If you want to keep the longer phrase, replace the last "replace by" with "with".  (Oof, ended the comment with a double preposition.  Sorry about that.)

Also, what about ``newChar`` instead of ``replacementChar``?